js/app/modules/card/list.js

```diff
@@ -30,6 +30,7 @@ const List = new Module.Class({
             true),
     },
 
+
```

Comment:
nitpick: Remove this blank line (otherwise this commit is good, so I'll remove the blank line myself and commit it)